### DISCUSSION TITLE
Reset variable reporter position

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#5526] [Ideas]
It would be nice to be able to reset the variable reporter position, like when the variable is first created it's reset, but you can't reset it afterwards. You could include it in the dropdown menu that appears when you click on the variable reporter, along with "hide, large readout, slider". I think this would be useful for organization purposes.

**Comment by DNin01** [comment]
This would also be useful for lists if they happen to move off the screen (there's a bug where if you drag the scroll bar on a list then hide and show the list, it can be moved outside the project player window).
